library/src/fy_library/mixins/property/mixin_import_map/using_parsed_fy_py_files_fy.py

```diff
@@ -39,7 +39,7 @@ class MixinImportMap_UsingParsedFyPyFiles_PropertyMixin(
     def _mixin_import_map(self) -> Dict[str, str]:
         # fy:end <<<===
         mixin_import_map = {
-            parsed_fy_py_file.template_model.entity_key: self.__parsed_file_python_import(
+            parsed_fy_py_file.entity_key: self.__parsed_file_python_import(  # type: ignore
```

Comment:
All you need to do is add an abstract property `entity_key` to the `ParsedFyPyFile` class. We need to do that anyway.